src/software/jetson_nano/services/network.cpp

```diff
@@ -30,12 +30,33 @@ std::tuple<TbotsProto::PrimitiveSet, TbotsProto::World> NetworkService::poll(
                                                                    world_msg};
 }
 
+float calculatePacketLossRate(std::queue<uint64_t> recent_proto_seq_nums,
+                              uint64_t seq_num)
```

Comment:
This function should be added to the `NetworkService` as a _private_ helper function. We can then add documentation for it in the header file as well.